packages/server/src/constants/index.ts

```diff
@@ -1,16 +1,15 @@
 import { constants, objectStore, roles } from "@budibase/backend-core"
 import {
-  FieldType as FieldTypes,
+  FieldType,
   INTERNAL_TABLE_SOURCE_ID,
   Table,
   TableSourceType,
 } from "@budibase/types"
 
 export {
```

Comment:
Do we actually need these exports? Why not using directly the ones from `types`, if we are already refactoring it?

Reply:
True - I'll finish getting rid of the exports!